mapclientplugins/geometryfitter/view/geometryfitterwidget.py

```diff
@@ -417,14 +417,17 @@ def _updateDisplayWidgets(self):
 
     def _displayErrors(self):
         rmsError, maxError = self._fitter.getDataRMSAndMaximumProjectionError()
+        element_id, value = self._fitter.getModelWorstElementJacobianInfo()
         rms_error_text = "-" if rmsError is None else f"{rmsError}"
         self._ui.displayRMSError_lineEdit.setText(rms_error_text)
         self._ui.displayRMSError_lineEdit.setCursorPosition(0)
         max_error_text = "-" if maxError is None else f"{maxError}"
         self._ui.displayMaxError_lineEdit.setText(max_error_text)
         self._ui.displayMaxError_lineEdit.setCursorPosition(0)
-        logger.info(f"RMS Error: {rms_error_text}")
-        logger.info(f"Max. Error: {max_error_text}")
+        model_quality = "good" if value > 0.5 else "fair" if value > 1e-07 else "poor" if value > -0.01 else "bad"
```

Comment:
Can't use arbitrary values for judgement here as don't know the scale of the model/units.
I suggest making the jacobian be det(gradient(model_coordinates, reference_model_coordinates)) so it always has a physically realistic relative value, and this can be written out here.
I don't like the qualitative value judgements; we can have a tool tip which explains zero/negative == bad, meaning of values. I *think* the jacobian should give the ratio of differential volumes, but would need to prove this.